consul/files/consul.json

```diff
@@ -1,6 +1,11 @@
 {
   {% if is_ui %}
-  "ui_dir": "{{ home_dir }}/dist",
+  "ui_dir": "{{ ui_install_path }}/dist",
+     {% if ui_public_target %}
+  "addresses" : {
+    "http": "{{ salt['network.interfaces']()['eth0']['inet'][0]['address'] }}"
```

Comment:
Agreed. Do you have any good ideas on how to do that? Maybe default to eth0 and then let it be configurable via a pillar?